roles/ceph-update/tasks/ceph_owner.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Isn;t this run at very end (last thing in site.yml). Is below work ok to be done that late in the flow?